flow-sample/src/main/java/com/example/flow/MainActivity.java

```diff
@@ -69,24 +76,30 @@ protected void onCreate(Bundle savedInstanceState) {
     containerAsBackTarget = (HandlesBack) container;
   }
 
+  @Override public Object onRetainNonConfigurationInstance() {
```

Comment:
Is it worth adding a suppress for the deprecation warning? Ideally with a link to jw's issue tracker thing.